Modelica/Mechanics/MultiBody/Examples/Loops/Utilities.mo

```diff
@@ -13,20 +13,20 @@ package Utilities "Utility models for Examples.Loops"
       "Offset of crank pin from center axis";
     parameter SI.Length crankPinLength=0.1
       "Offset of crank pin from center axis";
-    parameter SI.Angle cylinderInclination=0 "Inclination of cylinder";
+    parameter SI.Angle cylinderInclinationAngle=0 "Inclination of cylinder";
```

Comment:
Why this change?

Reply:
Oh. I did this 3 weeks ago.

Reply:
Ah! To avoid a name clash, see https://github.com/modelica/ModelicaStandardLibrary/pull/3091/files#diff-591434336330c10e1295030fd06a33c8R94.

Reply:
I see now. There will be a name conflict to a fixRotation instance. OK for me.